Doc/library/string.rst

```diff
@@ -588,6 +589,14 @@ The available presentation types for :class:`float` and
    |         | as altered by the other format modifiers.                |
    +---------+----------------------------------------------------------+
 
+The available presentation types for :class:`complex` are same as for
+:class:`float`.  Both real and imaginary components of a complex number are
+formatted as floating-point numbers, accordingly specified presentaion type.
+They separated by mandatory sign of the imaginary part, terminated by ``j``
+suffix.  If presentation type is missing --- result match the output of
+:func:`str` (numbers also surrounded by round parentheses, if real part is
```

Comment:
If by 'round parentheses' you mean the `()` characters, then 'round' is unnecessary. `()` are the only symbols named 'parentheses'.

There _is_ some ambiguity for other enclosures, though:

 - `{}` -- 'braces', 'curly braces', or 'curly brackets'
 - `[]` -- 'brackets', 'square brackets' or (rarely) 'square braces'
 - `<>` -- 'angle brackets'